css/logo-nav.css

```diff
@@ -69,15 +69,6 @@ animation-delay: 1.1s;
 a {
     color: black;
 }
-#nbviewer {
-	color:gray;
-    font-weight: 500;
-    text-decoration: underline;
-}
-#nbviewer:hover {
-    color: rgb(224, 103, 29);   
-}
-
```

Comment:
move below, with a generic rule " All ling in paragraphs in these section" instead of manually styling each links.